101-vm-with-rdp-port/azuredeploy.json

```diff
@@ -0,0 +1,255 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+
+    "parameters": {
+
+        "location": {
+            "type": "string",
+            "allowedValues": [
+                "West US",
+                "East US",
+                "West Europe",
+                "East Asia",
+                "Southeast Asia"
+            ],
+            "metadata": {
+                "description": "The region to deploy the resources into"
+            }
+        },
+
+        "publicDnsName": {
+            "type": "string",
+            "metadata": {
+                "description": "Unique public DNS prefix for the deployment. The fqdn will look something like '<dnsname>.westus.cloudapp.azure.com'. Up to 62 chars, digits or dashes, lowercase, should start with a letter: must conform to '^[a-z][a-z0-9-]{1,61}[a-z0-9]$'."
+            }
+        },
+
+        "storageAccountName": {
+            "type": "string",
+            "metadata": {
+                "description": "Name of the Storage Account. This has to be a uinique name, up to 24 chars, all lowercase."
+            }
+        },
+
+        "vmName": {
+			"type": "string",
+			"defaultValue": "vm",
+			"metadata": {
+				"description": "The name of the VM"
+			}
+		},
+
+        "imagePublisher": {
```

Comment:
Look at https://github.com/Azure/azure-quickstart-templates/blob/master/101-simple-windows-vm/azuredeploy.json and consider doing Windows version that way (simpler)